src/char/int_clan.cpp

```diff
@@ -33,6 +33,8 @@ struct clan* inter_clan_fromsql(int clan_id){
 	struct clan* clan;
 	char* data;
 	size_t len;
+ data = 0;
```

Comment:
This is within a struct. Ask Shaun how to validate these,, if they're any different than others.